### ISSUE TITLE
Fix up precision land flight mode parameters

### ISSUE THREAD

**Issue by hamishwillee** [PX4/PX4-user_guide#100]
> **NOTE: This isn't in master yet, so I will add a big fat note on the released docs and put on hold the rest of this for now** See https://github.com/PX4/Firmware/pull/8160

[Precision land](https://docs.px4.io/en/advanced_features/precland.html) was merged in #98 and updated in #99. As part of that the parameters were linked to the full parameter reference. But ...

@ndepal None of the parameters mentioned appear in the [parameter reference](https://docs.px4.io/en/advanced_config/parameter_reference.html) - e.g. LTEST_MODE, LTEST_SCALE_X, LTEST_SCALE_Y etc.

Can we get them added? Not only does this force proper documentation (exhaustive), but it means that all the links I just added to the reference will actually work :-0.

Actions
* [ ] Make sure that the parameter reference for all precland parameters is generated
* [ ] Update the precland docs to have tabular docs for the main relevant parameters (as per other flight modes)
* [ ] Add either dedicated flight mode doc for this mode, or more likely a link in the top level flight mode docs.
* [ ] Fix doc to make it clear that only the MarkOne beacon can be used. As per https://github.com/PX4/px4_user_guide/pull/99#discussion_r151933047

**Comment by ndepal** [comment]
I'm not sure what is required for the parameters to show up there.
I'm guessing one criterion is that the code needs to be in master, which it is no yet.
If the param reference is not generated from the `prarams.c`, please let me know what needs to be added.

**Comment by hamishwillee** [comment]
My fault - this should not be merged until in master. At that point the parameters if present should get picked up. I vote we look at this again once this is merged. IN the meantime I added a note on the topic so people won't try and use it yet!

**Comment by hamishwillee** [comment]
Closing. I believe this was all addressed.
